cmd/neofs-cli/modules/container.go

```diff
@@ -425,10 +425,10 @@ Container ID in EACL table will be substituted with ID from the CLI.`,
 			for i := 0; i < awaitTimeout; i++ {
 				time.Sleep(1 * time.Second)
 
-				eaclSig, err := cli.GetEACLWithSignature(ctx, id, globalCallOptions()...)
+				table, err := cli.GetEACL(ctx, id, globalCallOptions()...)
```

Comment:
I suggest to leave usage of `GetEACLWithSignature` method, but with signature check.

Reply:
`GetEACL` method encapsulates signature check inside. I don't think explicit signature check is an important thing there.
We have `GetEACLWithSignature` in `neofs-cli get-eacl` command if you want to provide use case example for this function.

Reply:
If we use GetEACLWithSignature, we still can output the signature event if it is wrong. But this is not necessary.

Reply:
> If we use GetEACLWithSignature, we still can output the signature event if it is wrong. But this is not necessary.

That is exactly how it's done in `get-eacl` command. As for `--await` flag in `set-eacl` I don't think it is necessary there.